tests/spec/smoketest-eq-services-spec.js

```diff
@@ -0,0 +1,68 @@
+const AuthorHome = require('../pages/AuthorHome.page');
+const CreateQuestionnairePage = require('../pages/CreateQuestionnaire.page');
+const DesignQuestionnairePage = require('../pages/DesignQuestionnaire.page');
+
+const BASE_URL = process.env.BASE_URL || 'http://localhost:3000';
+
+describe('eQ Services Smoke Test', () => {
+  beforeAll(() => {
+    browser.url(BASE_URL);
+    browser.click(AuthorHome.createQuestionnaireButton());
+    browser.setValue(CreateQuestionnairePage.questionnaireTitle(), 'title');
+    browser.setValue(CreateQuestionnairePage.questionnaireDescription(), 'description');
+    browser.selectByVisibleText(CreateQuestionnairePage.questionnaireTheme(), 'default');
+    browser.selectByVisibleText(CreateQuestionnairePage.questionnaireLegalBasis(), 'StatisticsOfTradeAct');
+    browser.click(CreateQuestionnairePage.navigationToggle());
+    browser.click(CreateQuestionnairePage.createButton());
+  });
+
+  it('should update navigation title when section title changed', () => {
+    browser.waitForExist("h3[class*='SectionTitle']");
```

Comment:
I don't think we should need `waitForExist` anywhere in these tests.
If you switch the style to a promise chain then you wont need to wait for events.
eg, instead of 
```
browser.click(CreateQuestionnairePage.navigationToggle());
browser.click(CreateQuestionnairePage.createButton());
```
You would have 
```
browser.click(CreateQuestionnairePage.navigationToggle())
.click(CreateQuestionnairePage.createButton());
```